rpi/gpio.go

```diff
@@ -59,20 +95,48 @@ func (gp gpioPin) SetPWMFreq(ctx context.Context, freqHz uint, extra map[string]
 	return gp.pi.SetPWMFreqBcom(gp.bcom, freqHz)
 }
 
+func (pi *piPigpio) reconfigureGPIOs(ctx context.Context, cfg *Config) error {
+	// Set new pins based on config
+	pi.gpioPins = map[int]*rpiGPIO{}
+	for _, newConfig := range cfg.Pins {
+		if newConfig.Type != rpiutils.PinGPIO {
+			continue
+		}
+		bcom, have := rpiutils.BroadcomPinFromHardwareLabel(newConfig.Pin)
+		if !have {
+			return errors.Errorf("no hw pin for (%s)", newConfig.Pin)
+		}
+		pin := &rpiGPIO{name: newConfig.Name, pin: bcom}
+		pi.gpioPins[int(bcom)] = pin
+	}
+	return nil
+}
+
```

Comment:
Can we write a test, or verify that the tests check that the lables "11" and "io0" are still interchangeable and useable as pin numbers in the pi driver code?